webapp/src/templates/partials/actionbar.html

```diff
@@ -99,11 +99,11 @@
             </a>
             <div class="dropdown-menu sub-actions" mm-auth="can_verify_reports" ng-show="actionBar.right.verifyingReport">
               <a class="mm-icon mm-icon-caption verify-error" ng-class="{ active: actionBar.right.verified === false }" ng-click="emit('VerifyReport', false)">
-                <span class="verify-icon" ng-include="'templates/partials/svg-icons/report-verify-invalid-icon.html'"></span>
+                <span class="verify-icon" ng-include="'templates/partials/svg-icons/error-icon.html'"></span>
                 <span translate>reports.verify.invalid</span>
               </a>
               <a class="mm-icon mm-icon-caption verify-valid" ng-class="{ active: actionBar.right.verified }" ng-click="emit('VerifyReport', true)">
-                <span class="verify-icon" ng-include="'templates/partials/svg-icons/report-verify-valid-icon.html'"></span>
+                <span class="verify-icon" ng-include="'templates/partials/svg-icons/ok-icon.html'"></span>
```

Comment:
I think the old names were better - they were more descriptive. Consider reverting this rename...